components/scream/src/dynamics/homme/atmosphere_dynamics.cpp

```diff
@@ -287,22 +293,42 @@ size_t HommeDynamics::requested_buffer_size_in_bytes() const
   }
   fv_phys_requested_buffer_size_in_bytes();
 
-  return fbm.allocated_size()*sizeof(Real);
+  const size_t homme_request = fbm.allocated_size()*sizeof(Real);
+
+  return homme_request + interface_request;
 }
 
 void HommeDynamics::init_buffers(const ATMBufferManager &buffer_manager)
 {
   EKAT_REQUIRE_MSG(buffer_manager.allocated_bytes() >= requested_buffer_size_in_bytes(),
                    "Error! Buffers size not sufficient.\n");
 
+  // Local memory used by interface
+  constexpr int N = HOMMEXX_PACK_SIZE;
+  using Pack = ekat::Pack<Real,N>;
+  const int nlevs = m_dyn_grid->get_num_vertical_levels();
+  const int npacks = ekat::PackInfo<N>::num_packs(nlevs);
+  Pack* packed_mem = reinterpret_cast<Pack*>(buffer_manager.get_memory());
+
+  m_buffer.otau = decltype(m_buffer.otau)(packed_mem, npacks);
```

Comment:
Do these buffers persist beyond one run_impl call? I thought they are supposed to hold only temporary data.

Reply:
@bartgol can weigh in, but in any case, I think a view that is written once at startup and then read repeatedly during time stepping should be kept as a simple View object in the HommeDynamics object.

Reply:
Wait, you are right. This is overwritten. I'll move it.